README.md

```diff
@@ -48,6 +48,8 @@ To integrate the SDK into a mobile app, follow these steps:
 
 > **Note**: You must use `gomobile bind` on a package you create, not directly on the SDK packages.
 
+The easiest way to integrate with the SDK in a mobile app is by using the [`x/mobileproxy` library](./x/mobileproxy/) 
```

Comment:
"easiest" might not be accurate? Especially once we had a platform wrapper library in the future.

```suggestion
A good example to integrate with the SDK in a mobile app is by using the [`x/mobileproxy` library](./x/mobileproxy/) 
```

Reply:
I meant the easiest way currently available.

I rephrased it to "An easy way". It's more than an example. That's what people will integrate with.